src/views/Login/Ticket.js

```diff
@@ -168,71 +227,58 @@ export default function Ticket({ className, goHome }) {
               <Grid.Item
                 full
                 as={TicketInput}
-                type={values.showTicket ? 'text' : 'password'}
+                accessory={<TicketInputAccessory name="ticketHidden" />}
+                hidden={values.ticketHidden}
+                labelAccessory={<AdvancedOptions />}
+                className="mt3"
                 name="shard1"
                 label="Shard 1"
               />
               <Grid.Item
                 full
                 as={TicketInput}
-                type={values.showTicket ? 'text' : 'password'}
+                accessory={<TicketInputAccessory name="ticketHidden" />}
+                hidden={values.ticketHidden}
+                className="mt3"
                 name="shard2"
                 label="Shard 2"
               />
               <Grid.Item
                 full
                 as={TicketInput}
-                type={values.showTicket ? 'text' : 'password'}
+                className="mt3"
+                accessory={<TicketInputAccessory name="ticketHidden" />}
+                hidden={values.ticketHidden}
                 name="shard3"
                 label="Shard 3"
               />
             </Condition>
 
-            <Grid.Item full as={CheckboxInput} name="showTicket" label="Show" />
-
             <Condition when="usePassphrase" is={true}>
               <Grid.Item
                 full
                 as={PassphraseInput}
+                className="mt3"
                 name="passphrase"
                 label="Wallet Passphrase"
               />
             </Condition>
 
             <Grid.Item full as={FormError} />
 
-            <Grid.Item full as={SubmitButton} handleSubmit={handleSubmit}>
+            <Grid.Item
+              full
+              as={SubmitButton}
+              center
+              handleSubmit={handleSubmit}>
               {isWarning =>
```

Comment:
Maybe this is as good a time as any to tear this out. See also #405. (But we might want further discussion on that.)

Reply:
Yeah while I was styling this, I felt like it was an unnecessary layer of abstraction, but what do I know lol